client/src/app/auth/reset-password/page.tsx

```diff
@@ -0,0 +1,161 @@
+"use client";
+import { useState, useEffect } from "react";
+import { useRouter, useSearchParams } from "next/navigation";
+import useFetch from "@/src/hooks/useFetch";
+import { AuthService } from "@/src/services/AuthService";
+import Navbar from "@/src/components/Navbar/Navbar";
+import Footer from "@/src/components/Footer/Footer";
+import useThemeStore from "@/src/store/themeStore";
+import Image from "next/image";
+import ResetPasswordSvg from "@/public/assets/Login.svg";
+import Link from "next/link";
+
+const ResetPassword = () => {
+  const router = useRouter();
+  const searchParams = useSearchParams();
```

Comment:
In Next.js we can get searchParams directly in props also. It will work for both it's a another way to do it if you don't want to write more line of code as it will work for both.